dev-perl/Array-Diff/Array-Diff-0.70.0-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+DIST_AUTHOR=TYPESTER
+DIST_VERSION="0.07"
+
+inherit perl-module
+
+DESCRIPTION="Find the differences between two arrays"
+LICENSE="|| ( Artistic GPL-2 )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test +xs"
```

Comment:
Repoman will have told you this requires entries into `metadata.xml`, as `xs` is not a globally defined `USE`.

Consider borrowing the definition from `dev-perl/List-MoreUtils::gentoo`'s `metadata.xml`